sparse_ho/algo/implicit.py

```diff
@@ -13,11 +13,11 @@ class Implicit():
     Parameters
     ----------
     max_iter: int (default=100)
-        maximum number of iteration for the inner solver
+        Maximum number of iteration for the inner solver
     max_iter_lin_sys: int (default=100)
-        maximum number of iteration for the resolution od the linear system
+        Maximum number of iteration for the resolution od the linear system
```

Comment:
of (od) + iterations +  missing dot at the end of sentences in this docstring.

Reply:
done